packages/client/src/base/command-stack.ts

```diff
@@ -15,25 +15,37 @@
  ********************************************************************************/
 import {
     CommandExecutionContext,
+    CommandExecutionData,
     CommandStack,
     Disposable,
     DisposableCollection,
     Event,
     GModelRoot,
     ICommand,
-    LazyInjector,
-    SetModelCommand,
-    UpdateModelCommand
+    ICommandStack,
+    LazyInjector
 } from '@eclipse-glsp/sprotty';
-import { inject, injectable, preDestroy } from 'inversify';
+import { inject, injectable, postConstruct, preDestroy } from 'inversify';
+import { Emitter } from 'vscode-jsonrpc';
```

Comment:
We definitely want the import from `@eclipse-glsp/sprotty` and not `vscode-jsonrpc`.

Reply:
Fair point